sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/education-program-offering.institutions.createOffering.e2e-spec.ts

```diff
@@ -0,0 +1,161 @@
+import {
+  Institution,
+  InstitutionLocation,
+  OfferingIntensity,
+  OfferingStatus,
+  OfferingTypes,
+  User,
+} from "@sims/sims-db";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  createFakeInstitutionLocation,
+  OfferingDeliveryOptions,
+} from "@sims/test-utils";
+import {
+  createTestingAppModule,
+  BEARER_AUTH_TYPE,
+  InstitutionTokenTypes,
+  getInstitutionToken,
+  authorizeUserTokenForLocation,
+  getAuthRelatedEntities,
+  createFakeEducationProgram,
+} from "../../../testHelpers";
+import * as request from "supertest";
+
+describe("EducationProgramOfferingInstitutionsController(e2e)-createOffering", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  let collegeF: Institution;
+  let collegeFLocation: InstitutionLocation;
+  let collegeFUser: User;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    const { institution, user: institutionUser } = await getAuthRelatedEntities(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+    );
+    collegeF = institution;
+    collegeFUser = institutionUser;
+    collegeFLocation = createFakeInstitutionLocation(collegeF);
+    await authorizeUserTokenForLocation(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+      collegeFLocation,
+    );
+  });
+
+  it("Should create a new offering when passed valid data.", async () => {
+    // Arrange
+    const institutionUserToken = await getInstitutionToken(
+      InstitutionTokenTypes.CollegeFUser,
+    );
+    const fakeEducationProgram = createFakeEducationProgram(
+      collegeF,
+      collegeFUser,
+    );
+    fakeEducationProgram.sabcCode = "nbgvc";
+    const savedFakeEducationProgram = await db.educationProgram.save(
+      fakeEducationProgram,
+    );
+    const endpoint = `/institutions/education-program-offering/location/${collegeFLocation.id}/education-program/${savedFakeEducationProgram.id}`;
+    const payload = {
+      offeringName: "Offering 1",
+      yearOfStudy: 1,
+      showYearOfStudy: true,
+      offeringIntensity: OfferingIntensity.fullTime,
+      offeringDelivered: OfferingDeliveryOptions.Onsite,
+      hasOfferingWILComponent: "no",
+      studyStartDate: "2023-09-01",
+      studyEndDate: "2024-06-30",
+      lacksStudyBreaks: false,
+      studyBreaks: [
+        {
+          breakStartDate: "2023-12-01",
+          breakEndDate: "2024-01-01",
+          breakDays: 32,
+          eligibleBreakDays: 21,
+          ineligibleBreakDays: 11,
+        },
+      ],
+      studyBreaksOverlappingDetection: false,
+      studyPeriodBreakdown: {
+        totalDays: 304,
+        totalFundedWeeks: 42,
+        fundedStudyPeriodDays: 293,
+        unfundedStudyPeriodDays: 11,
+      },
+      offeringType: OfferingTypes.Public,
+      offeringDeclaration: true,
+      mode: "editable",
```

Comment:
I believe `mode`, `studyBreaksOverlappingDetection`, `validationWarnings`, `validationInfos`, `applicationId`, and `applicationStatus` are not part of the DTO and can be removed.

Reply:
I still can see properties like `studyBreaksOverlappingDetection`, `validationWarnings`, and `validationInfos` in the payloads. Are those required?